model/human_hand.urdf.xacro

```diff
@@ -145,6 +145,11 @@
 				<xacro:property name="side" value="${mapping.get('handedness','right')}"/>
 			</xacro:if>
 			<xacro:property name="layout" value="${mapping['layout']}"/>
+			<!-- As we share taxel meshes across all layouts, but L1 doesn't have some taxel indices,
+			     we map L1 taxel indices onto corresponding L2/L3 taxel indices here -->
+			<xacro:if value="${layout == 'L1'}">
+				${mapping.update([(k, mapping.get(v,-1)) for k, v in [('tmm', 'tmo'), ('ptmp', 'ptop'), ('ptmd', 'ptod')]])}
```

Comment:
```suggestion
				<xacro:if value="${mapping.update([(k, mapping.get(v,-1)) for k, v in [('tmm', 'tmo'), ('ptmp', 'ptop'), ('ptmd', 'ptod')]])}" />
```
The original code in this PR produces a "None" in the middle of the urdf. With the suggested code change there is no additional line, but the code is nicely evaluated. 
FYI, a xacro dummy property did not work as the value was lazy evaluated if not called and one does not want to use a dummy property, so the xacro:if was the best I found to "swallow" the None.